internal/handlers/pg/pgdb/query.go

```diff
@@ -173,13 +174,41 @@ func buildQuery(ctx context.Context, tx pgx.Tx, sp *SQLParam) (string, error) {
 	}
 	q += `FROM ` + pgx.Identifier{sp.DB, table}.Sanitize()
 
+	if sp.SqlFilters != nil {
+		q = appendSqlFilters(q, sp.SqlFilters)
+	}
+
 	if sp.Explain {
 		q = "EXPLAIN (VERBOSE true, FORMAT JSON) " + q
 	}
 
 	return q, nil
 }
 
+// appendSqlFilters adds WHERE clause with given filters to the query.
+func appendSqlFilters(q string, sqlFilter *types.Document) string {
+	var filters []string
+
+	for k, v := range sqlFilter.Map() {
+		switch k {
+		case "_id":
+			switch v := v.(type) {
+			case *types.ObjectID:
+				filters = append(filters, fmt.Sprintf(`(_jsonb->'_id')::jsonb->>'$o' = '%s'`, must.NotFail(fjson.Marshal(v))))
```

Comment:
We should use placeholders instead of `%s`

Reply:
Done.